src/about/AboutScreen.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { View,
+          Text,
+          StyleSheet,
+          TouchableOpacity,
+          Linking,
+        } from 'react-native';
+import Logo from './Logo';
```

Comment:
Why not import Logo from 'common' dir here? I see that you wanted to use a different value of margin for the logo, but creating a brand new component with just one different line of code doesn't sound like a very good idea. Don't you think?